dns/dnssec.py

```diff
@@ -886,16 +640,15 @@ def _make_rrsig_signature_data(
 
 
 def _make_dnskey(
-    public_key: PublicKey,
+    public_key: Union[PublicKey],
```

Comment:
This looks like the result of multiple refactorings, and shouldn't be needed.

Reply:
Indeed, fixed in 65babe74ee1363c0ca797845dc737e21e55bb18a